ote_sdk/ote_sdk/entities/label_schema.py

```diff
@@ -540,9 +478,37 @@ def get_group_containing_label(self, label: LabelEntity) -> Optional[LabelGroup]
         return None
 
     def get_labels_exclusive_to(self, label: LabelEntity) -> List[LabelEntity]:
-        """Returns a list of labels that are exclusive to the passed label"""
-        self.__get_label(label)
-        return list(self.exclusivity_graph.neighbors(label))
+        """Returns a list of labels that are exclusive to the passed label."""
+        return self.__get_exclusivity_recursion(label, True)
+
+    def __get_exclusivity_recursion(
+        self, label, add_children: bool = False
```

Comment:
Since I was thinking of a wrong logic, maybe we can remove the add_children altogether.

Reply:
Done